tests/openvino/test_modeling.py

```diff
@@ -1296,12 +1302,15 @@ def test_compare_to_transformers(self, model_arch):
         ov_outputs = ov_model(**tokens)
         self.assertTrue("logits" in ov_outputs)
         self.assertIsInstance(ov_outputs.logits, torch.Tensor)
-        self.assertTrue("past_key_values" in ov_outputs)
-        self.assertIsInstance(ov_outputs.past_key_values, tuple)
-        is_stateful = ov_model.config.model_type not in not_stateful
-        self.assertEqual(ov_model.stateful, is_stateful)
-        if is_stateful:
-            self.assertTrue(len(ov_outputs.past_key_values) == 1 and len(ov_outputs.past_key_values[0]) == 0)
+        if model_arch in self.SUPPORTED_SSM_ARCHITECTURES:
+            self.assertTrue("cache_params" in ov_outputs)
```

Comment:
shouldn't we do the same checks in the else branch here as well ?

Reply:
No, we should not. That is because we have transformers in else-branch. The name of KV cache outputs are different.
For Mamba based models cache output name is "cache_params".

Reply:
Thank you for the explanation. Added more checks.